validation/src/main/scala/hmda/validation/engine/lar/quality/LarQualityEngine.scala

```diff
@@ -7,6 +7,11 @@ import hmda.validation.rules.lar.quality._
 
 trait LarQualityEngine extends LarCommonEngine with ValidationApi {
 
+  private def q022(lar: LoanApplicationRegister): LarValidation = {
+    // TODO: 2017 is hardcoded for now.  It will be passed in through the API at a later date
```

Comment:
Yes, I remember discussing that it was going to come from the front end (though hadn't carefully read the issue itself, so that was a helpful pointer). And I know they're not currently hooked up yet. So perhaps a better way to phrase the question is, what (if anything) blocks us from doing all of the back-end part now?

(I've already asked separately about what's blocking integration of front and back end, which is outside the scope of this issue.  I don't see that question as blocking this one, though.)